engine/common/api_helpers/utils.py

```diff
@@ -52,34 +50,6 @@ def __repr__(self):
         return "%s()" % self.__class__.__name__
 
 
-class URLValidatorWithoutTLD(URLValidator):
```

Comment:
I remember I added this because some OSS users wanted to make requests to docker containers, which default url validator didn't permit. Do we have url validation for webhooks url? if so, should we use something like that there:
               ```
 if live_settings.DANGEROUS_WEBHOOKS_ENABLED:
                    URLValidatorWithoutTLD()(webhook)
                else:
                    URLValidator()(webhook)
```

Reply:
Correct DANGEROUS_WEBHOOKS_ENABLED settings is handled there.  Validation happens later in the process now since URLs can be templates